src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");

```

Comment:
The message should be more informative and indicate that it is an 'Excavator' that is working. Consider rephrasing it to include the machine type, for example: 'Excavator is working.'